Chess/ProjectSettings/ProjectSettings.asset

```diff
@@ -748,10 +748,10 @@ PlayerSettings:
   apiCompatibilityLevel: 6
   activeInputHandler: 2
   windowsGamepadBackendHint: 0
-  cloudProjectId: 42a3e924-4bd9-41f8-adc9-59b18a032599
+  cloudProjectId: aa783631-e8b8-46c6-9741-09566c8e26c2
```

Comment:
Could we delete `cloudProjectId` and `projectName` from the settings so that checking this repo out forces the person to configure those values in the services window in the editor?

Reply:
Good idea